external/fv3fit/docs/usage.rst

```diff
@@ -17,6 +17,7 @@ It has several useful features:
 
 * Different architectures: RNN, dense, etc...also extensible.
 
+* Any artifacts created during training saved in the `artifacts` directory will be copied to the final output location along with the trained model.
```

Comment:
This isn't specific to TransformedModel, is it? Can you put this documentation wherever `fv3fit.train` is documented?

Reply:
Whoops, saw the doc was named 'usage' and assumed it was general usage. I'll fix this.